### PR TITLE
Comment about develop branch

### LINKED ISSUE #169: Comment about checking develop branch?
In the exercise, [Exercise: Improve Code Style of Our Project](https://carpentries-incubator.github.io/python-intermediate-development/15-coding-conventions/index.html#exercise-improve-code-style-of-our-project), it has them checkout the development branch before creating the feature branch.  This isn't really needed since at this point the develop branch and main are pointing at the same commit.  Should we add a comment to clarify that?

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #169.

### CODE REVIEW

File: _episodes/15-coding-conventions.md [hunk 1]
```diff
@@ -480,16 +484,17 @@ because an incorrect comment causes more confusion than no comment at all.
 >>     By convention, variable names should be in lowercase with optional underscores
 >>     so you should rename the variable 'InFiles' to, e.g., 'infiles' or 'in_files'.
 >>
->>  3. There is an extra blank line on line 20 in `inflammation-analysis.py`.
+>>  3. There are 2 blank lines starting from line 19 in `inflammation-analysis.py`.
```
Comment (anenadic) [inline _episodes/15-coding-conventions.md:487]:
```suggestion
>>  3. There are two blank lines starting from line 19 in `inflammation-analysis.py`.
```

File: _episodes/15-coding-conventions.md [hunk 2]
```diff
@@ -420,8 +420,12 @@ because an incorrect comment causes more confusion than no comment at all.
 
 > ## Exercise: Improve Code Style of Our Project
 > Let's look at improving the coding style of our project.
-> First create a new feature branch called `style-fixes` off our `develop` branch
-> and switch to it (from the project root):
+> First, from the project root, create a new feature branch called `style-fixes` off our `develop` branch
+> and switch to it.
```
[the remark this answers is not in the document]
Comment (anenadic) [reply _episodes/15-coding-conventions.md:424]: Sounds good - will fix @douglowe

Comment (anenadic) [inline _episodes/15-coding-conventions.md:424]: > First, from the project root use `git switch` to create a new feature branch called `style-fixes` from our `develop` branch.

[the remark this answers is not in the document]
Comment (anenadic) [reply _episodes/15-coding-conventions.md]: Fixed via c9c9abd

**Review by douglowe — APPROVED** [review]
Looks good to me :)

**Pull request merged**